qa/tasks/cephfs/test_nfs.py

```diff
@@ -428,12 +428,14 @@ def test_cluster_info(self):
         '''
         self._test_create_cluster()
         info_output = json.loads(self._nfs_cmd('cluster', 'info', self.cluster_id))
+        info_ip = info_output[self.cluster_id][0].pop("ip")
         host_details = {self.cluster_id: [{
             "hostname": self._sys_cmd(['hostname']).decode("utf-8").strip(),
-            "ip": list(set(self._sys_cmd(['hostname', '-I']).decode("utf-8").split())),
             "port": 2049
             }]}
+        host_ip = self._sys_cmd(['hostname', '-I']).decode("utf-8").split()
         self.assertDictEqual(info_output, host_details)
+        self.assertTrue(any([ip in info_ip for ip in host_ip]))
```

Comment:
better I think would be:

```suggestion
        self.assertIn(ip, host_ip)
```

?

Reply:
Oh, maybe you want to do substring matching too (nested "in")? In that case maybe your construction is better.

Reply:
Here both `info_ip` and `host_ip` are list. They don't have any nested list. I just wanted to check if any of `host_ip` list values are in the `info_ip` list.  `assertIn()` would consider `info_ip` list as a single element and check in `host_ip`.